types/src/v0/impls/header.rs

```diff
@@ -1612,23 +1490,23 @@ mod test_headers {
             .await
             .unwrap()
             .0;
-        validate_proposal(
-            &proposal_state,
-            genesis.instance_state.chain_config,
-            &parent_leaf,
-            &proposal.clone(),
-            &vid_common,
-        )
-        .unwrap();
 
-        assert_eq!(
-            proposal_state.block_merkle_tree.commitment(),
-            proposal.block_merkle_tree_root()
-        );
+        // ValidatedTransition::new(
```

Comment:
Is this intentionally left commented out?

Reply:
Humm I thought this was previously commented in relation to the above TODO... Maybe I got confused will double check.